ledger-api/grpc-definitions/com/digitalasset/ledger/api/v1/command_completion_service.proto

```diff
@@ -62,6 +61,8 @@ message CompletionStreamRequest {
   LedgerOffset offset = 4;
 }
 
+// completion stream responses always start with the snapshot of
+// active set of commands and subsequently only updates to that ACS
```

Comment:
Please do not use the ACS to describe completions, it is confusing.
What is active set of commands exactly and what are the guarantees with regards to HA and DR?